ovn_cluster.sh

```diff
@@ -446,6 +446,39 @@ function start-db-cluster() {
     $SSL_ARGS start_northd
 }
 
+function check_etcd_mode() {
+    if [[ ! -z "$use_etc" ]]; then
+	    true
+    else
+        false
+    fi
+}
+
+function start_etcd() {
+    name=`hostname -s`
+
+    rm -rf ${name}.etcd
+
+    ip=`hostname -i | awk '{print $2}'`
+
+    etcd --name ${name}  --initial-advertise-peer-urls http://${ip}:2380 \
+     --max-txn-ops 1000000 \
+     --max-request-bytes 17179869184 \
+     --listen-peer-urls http://${ip}:2380 \
+     --listen-client-urls http://${ip}:2379,http://127.0.0.1:2379 \
+     --advertise-client-urls http://${ip}:2379 \
+     --initial-cluster-token etcd-cluster-1 \
+     --initial-cluster ovn1=http://10.126.111.40:2380,ovn2=http://10.126.111.52:2380,ovn3=http://10.126.111.66:2380 \
```

Comment:
Please make sure to look at ovn_cluster.sh fo use variables instead of hard coded ips.